src/Netatmo/IAirClient.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+using Netatmo.Models.Client;
+using Netatmo.Models.Client.Air;
+
+namespace Netatmo
+{
+    public interface IAirClient
+    {
+        Task<DataResponse<GetHomeCoachsData>> GetHomeCoachsData();
```

Comment:
Can you add `device_id` entry parameter like we do in WeatherClient ?
```suggestion
        Task<DataResponse<GetHomeCoachsData>> GetHomeCoachsData(string deviceId = null);
```